_examples/golang-sse/webapp/src/rpc.gen.ts

```diff
@@ -486,8 +485,13 @@ export interface WebrpcStreamOptions<T> extends WebrpcOptions {
   onOpen?: () => void;
   onClose?: () => void;
 }
+
 export interface WebrpcOptions {
   headers?: HeadersInit;
-  signal?: AbortSignal;
+  controller?: AbortController;
 }
 
+export interface WebrpcStreamController {
+  abort: (reason?: any) => void;
+  closed: Promise<void>;
```

Comment:
I renamed "wait" to "closed", as Vojtech mentioned. I think its a better name and still useful to have around.